src/web/mjs/engine/Connector.mjs

```diff
@@ -745,9 +745,56 @@ export default class Connector {
     }
 
     getFormatRegex() {
-        return{
-            chapterRegex:/\s*(?:^|ch\.?|ep\.?|chapter|chapitre|episode|#)\s*([\d.?\-?v?]+)(?:\s|:|$)+/i, // $ not working in character groups => [\s\:$]+ does not work
-            volumeRegex:/\s*(?:vol\.?|volume|tome)\s*(\d+)/i
-        };
+        if (this.tags.includes('vietnamese')) {
```

Comment:
Avoid if-else smell
Either merge this into a single regex, or place each language specific regex into the corresponding connector (object oriented approach)

Reply:
i was trying to avoid
```js
            chapterRegex: /\s*(?:^|ch\.?|ep\.?|chapter|chapitre|Bölüm|Chap|Chương|ตอนที่|Kapitel|Capitolo|Rozdział|Глава|Cap[ií]tulo|cap|episode|#):?\s*([\d.?\-?v?]+)(?:\s|:|$)+/i, // $ not working in character groups => [\s\:$]+ does not work
            volumeRegex: /\s*(?:vol\.?|volume|Sezon|Том|Band|Cilt|tome)\s*(\d+)/i
```
also didn't want to add it to every connector manually

Reply:
if the language was separate from the tags i could use Switch case, but that is not the case